torch/cuda/__init__.py

```diff
@@ -111,8 +111,8 @@ def _maybe_exchange_device(device: int) -> int:
 
 
 has_half: bool = True
-# Global variables dynamically populated by native code
-has_magma: bool = False
+has_magma: bool = getattr(torch._C, "_cuda_hasMagma", lambda: False)()
```

Comment:
This is meh
You can put this function in the general binding (instead of the cuda-only one) if you always need it.

Reply:
I was about to object, that Module.cpp should export no CUDA specific symbols, but as it exports `_has_cuda`, then I don't see why not do the same for magma